src/olympia/abuse/models.py

```diff
@@ -1161,3 +1166,13 @@ def notify_reviewer_decision(
                 **target_url_override,
             },
         )
+
+
+class CinderAppealText(ModelBase):
```

Comment:
it came out of a slack discussion about the scope of the changes strictly needed. https://mozilla.slack.com/archives/C06JB499BEC/p1719494590568029

We unfortunately can't just add it on the `CinderJob` because Cinder can combine appeals on multiple `CinderDecision`s for the same entity (e.g. Add-on), and can't add it to `CinderDecision` as there's a chance that more than one reporter appeals a decision. (Plus it'd be wasteful as most jobs and decisions aren't appealing related)

Reply:
Okay, that partially answers the question but still on the naming side why not name it `CinderAppeal` which would prevent a lot of future refactoring in the event we expand the scope and doesn't (seem to) impact it at all otherwise.?